BookingsApi/BookingsApi.UnitTests/Domain/EndPoints/UpdateDisplayNameTests.cs

```diff
@@ -13,19 +13,42 @@ public void Initialise()
         }
 
         [Test]
-        public void Should_update_display_name()
+        public async Task Should_update_display_name()
         {
             var newDisplayName = "New DisplayName";
+            var originalUpdatedDate = _endpoint.UpdatedDate;
+
+            await ApplyDelay();
```

Comment:
Could use the existing `SetProtected` to achieve the same result
```suggestion
            _endpoint.SetProtected(nameof(_endpoint.UpdatedDate), DateTime.UtcNow.AddMinutes(-1));
            var originalUpdatedDate = _endpoint.UpdatedDate;
```

Reply:
good shout, thanks :) have made this change